File: app/src/main/java/com/vn/ezlearn/models/Reading.java
package com.vn.ezlearn.models;

import java.util.List;

/**
 * Created by Windows 10 Version 2 on 10/10/2017.
 */

public class Reading {
    public Integer id;
    public String content;
    public String category_id;
    public String file_image;
    public String status;
    public String type;
    public String parent_id;
    public String created_at;
    public String updated_at;
    public String weight;
    public String file_audio;
    public String is_listening;
    public List<Content> questions;
}

File: app/src/main/java/com/vn/ezlearn/models/Reading.kt
package com.vn.ezlearn.models

/**
 * Created by Windows 10 Version 2 on 10/10/2017.
 */

class Reading {
    var id: Int? = null
    var content: String = ""
    var category_id: String? = null
    var file_image: String? = null
    var status: String? = null
    var type: String? = null
    var parent_id: String? = null
    var created_at: String? = null
    var updated_at: String? = null
    var weight: String? = null
    var file_audio: String? = null
    var is_listening: String? = null
    var questions: List<Content>? = null
}
